skins/oasis/js/Tracking.js

```diff
@@ -259,6 +259,16 @@ jQuery(function ($) {
 		}
 	});
 
+	// TODO remove after XW-2226 is done
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1135)